### DISCUSSION TITLE
'Hubs Cloud Offline' - still hasn't auto reset

### DISCUSSION THREAD

**Discussion by gantttt** [mozilla/hubs#3071] [Hubs Cloud Questions] [answered]
Hi all! 

My Hubs Cloud hosted on AWS went offline a few days ago, and still hasn't seem to reset/bounced back.

Getting the 'Hubs Cloud is currently offline. Check back shortly' splash page at both the site and the admin panel -- would be super grateful if anyone could point me in the right direction -- been going thru the docs, but i'm stumped!

many thanks - m

**Comment by robinkwilson** [comment, marked as answer]
**Quick question:**
Did you "update the stack" and set your instance to Offline mode? https://hubs.mozilla.com/docs/hubs-cloud-aws-updating-the-stack.html
Then did you "update the stack" and set your instance back to Online mode?

Otherwise:

**Check the # of EC2 instances**
If you're running Enterprise multi-server EC2 instances, look in AWS console > EC2 and see if you have the correct # of EC2 instances. If not, we'll have to "update the stack" to spin them back up.

**Restart your EC2 instances** 
AWS console > EC2 > click on your HC server(s) then "Terminate" instances. New ones will start up immediately after.

**Reply by gantttt** [reply]
thanks so much! i haven't updated the stack or touched anything, but going to check into that now as a way of resetting.

otherwise, seems like restarting the EC2 instance seems like the way to go? Just to be sure -- should I choose 'terminate instance' or 'reboot instance' (apologies if this is common knowledge/obvious too - im a bit out of my depth!)

**Reply by robinkwilson** [reply]
All good! "Terminate" is the correct one! Yes try this one first :)

**Reply by gantttt** [reply]
THERE GOES MY HERO - thx! terminating now + crossing fingers! 🤞

https://www.youtube.com/watch?v=EqWRaAF6_WY

**Reply by gantttt** [reply]
w00! one last quick note to say that that did the trick - just got back up and running - serious thx again

**Reply by nupur711** [reply]
How to reboot ec2 instance?
